tests/test_callables.py

```diff
@@ -0,0 +1,9 @@
+from typing import TYPE_CHECKING
+
+from basedtyping.callables import Function
+from basedtyping.type_checks import assert_type
+
+if TYPE_CHECKING:
```

Comment:
Why is this defined in type checking?

Oh, these are only tested within mypy, not pytest?

Might be better to document that.

Reply:
Down the line would be better to make it an explicit pytest instead of just a mypy check.

Reply:
> Down the line would be better to make it an explicit pytest instead of just a mypy check.

how would i do that?